api/swagger-spec/swagger.yml

```diff
@@ -52,6 +52,26 @@ paths:
           description: unexpected error
           schema:
             $ref: "#/definitions/error"
+  /v3/doctor/{train}/{release}/{uuid}:
```

Comment:
we should somehow know what version doctor info the user is sending? if not here we should include that in doctorInfo struct ?